AutoCollection/Exceptions.ts

```diff
@@ -36,9 +36,8 @@ class AutoCollectExceptions {
             var self = this;
             if (!this._exceptionListenerHandle) {
                 var handle = (reThrow: boolean, error: Error) => {
-                    var data = AutoCollectExceptions.getExceptionData(error, false);
-                    var envelope = this._client.getEnvelope(data);
-                    this._client.channel.handleCrash(envelope);
+                    this._client.trackException({exception: error});
+                    this._client.flush();
```

Comment:
`flush` isn't equal here, right? It's not using the `isNodeCrashing` parameter to `channel.triggerSend`

Reply:
Addressed, changed flush signature to accommodate the scenario.